src/metabase/driver/sql_jdbc.clj

```diff
@@ -162,6 +163,24 @@
                                :dialect (sql.qp/quote-style driver)))]
     (driver-api/execute-write-sql! db-id sql)))
 
+(defmethod driver/rename-tables! :sql-jdbc
```

Comment:
I don't think we should implement this for the base driver. 

Why? Because even though this will be valid SQL for most databaes, it might not be atomic.

What I suggest doing instead is implementing this for some other keyword, maybe something like `:sql-rename-supports-tx`, and having drivers derive from this keyword where relevant.